src/components/structures/auth/Registration.js

```diff
@@ -323,6 +323,14 @@ module.exports = React.createClass({
         });
     },
 
+    onEditServerDetailsClick(ev) {
+        ev.preventDefault();
+        ev.stopPropagation();
```

Comment:
nit: not sure why you need this? Is there some higher up click handler you don't want to get fired?

Reply:
It's not required here, but I had noticed many click handlers throughout the app that call `stopPropagation` anyway, so I assumed it was code style.